com.inventwithpython.flippy.json

```diff
@@ -2,15 +2,15 @@
     "id": "com.inventwithpython.flippy",
     "runtime": "org.freedesktop.Platform",
     "sdk": "org.freedesktop.Sdk",
-    "runtime-version": "20.08",
+    "runtime-version": "22.08",
```

Comment:
Can you try 23.08?

Reply:
Done in https://github.com/flathub/com.inventwithpython.flippy/pull/7/commits/238485aa6365949f0607a62f9854cf5b0d96bc8f